### ISSUE TITLE
求问seq2seq_attention训练显示acc问题

### ISSUE THREAD

**Issue by imshxz** [shibing624/pycorrector#45]
求问，想在seq2seq_att训练过程中显示acc，于是在seq2seq_attn_model.py中修改了	
`model.compile(optimizer=Adam(1e-3), metrics=['accuracy'])`
但是训练工程中仍未显示acc，val_acc。

`2019-01-11 09:42:54.750493: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
Epoch 1/40
137/137 [==============================] - 135s 987ms/step - loss: 0.1134 - val_loss: 0.3633
Epoch 2/40
137/137 [==============================] - 132s 966ms/step - loss: 0.1064 - val_loss: 0.5050
Epoch 3/40
137/137 [==============================] - 132s 963ms/step - loss: 0.1033 - val_loss: 0.5974
Epoch 4/40
137/137 [==============================] - 131s 959ms/step - loss: 0.1013 - val_loss: 0.4981
`

求助怎么办……非常感谢！

**Comment by shibing624** [comment]
seq2seq模型无准确率指标。

**Comment by imshxz** [comment]
感谢您的回复~您说的seq2seq无准确率指标是因为语言组织的多样性所以没有办法很好的衡量该模型的准确性吗？但是结果与我们训练数据的“参考答案”相比较还是会有个准确率输出？
我在seq2seq模型的seq2seq_model.py中修改了：model.compile(optimizer='rmsprop', loss='categorical_crossentropy', metrics=['accuracy'])，结果在训练中是可以显示acc和val_acc的
。但是我用同样的方法在seq2seq_attention模型中的seq2seq_attn_model.py修改就不可行。不明白为什么以及该怎么办了~
![image](https://user-images.githubusercontent.com/38749730/51287645-75971280-1a33-11e9-8353-7b8a11fa514b.png)

**Comment by zhazl** [comment]
同样有这个问题，@imshxz 有解决吗？
